packages/services/f-feature-management-vue3/src/createFeatureManagementForVue3.js

```diff
@@ -0,0 +1,45 @@
+import createFeatureManagement from '@justeat/f-feature-management';
+import { reactive } from '@vue/reactivity';
+
+/**
+ * Sets a property called `reactive` onto the FeatureManagement proxy, with a `getValue()` function on it.
+ * @param {object} featureManagement The reactive FeatureManagement proxy.
+ */
+function setReactiveProperty (featureManagement) {
+    if (featureManagement) {
+        featureManagement.reactive = {
+            getValue: key => featureManagement.getValue(key)
+        };
+    }
+}
+
+/**
+ * Gets a FeatureManagement object, swaps it for a Vue3 proxy, adds a `reactive` property
+ * then updates the `reactive` property every time new config is loaded.
+ * @param {object} settings.  See f-feature-management module docs.
+ * @returns Vue3 proxy for FeatureManagement
+ */
+async function createFeatureManagementForVue3 (settings) {
+    const originalOnUpdated = settings.onUpdated;
+
+    const overriddenSettings = {
+        ...settings,
+        onUpdated: () => {
+            if (typeof originalOnUpdated === 'function') {
+                originalOnUpdated();
+            }
+
+            /* eslint-disable no-use-before-define */
```

Comment:
Are these eslint comments necessary now?

Reply:
I couldn't work out a better way to do it TBH.

Reply:
I mean it looks like the function _is_ defined before being used (I'm assuming that it wasn't when these comments were added), so I was wondering if these were still required

Reply:
it's referring to `featureManagement`, which is defined lower down.

Reply:
Actually it was finding me a genuine bug 🤦 .  Have fixed now.